File: python/calculate_mode.py
def calculate_mode(lst):
    answer=[]
    frequency=0
    my_dict={}
    for i in lst:
        instances=lst.count(i)
        my_dict[i]=instances
        if instances > frequency:
            frequency= instances
    for i in my_dict:
        if my_dict[i]== frequency:
            answer.append(i)
    return answer

File: js/calculateMode.js
exports.calculateMode = function(lst) {
    const getCount=(ltr, word)=>{
        let count=0
        for(let i=0; i<word.length; i++){
            if(word[i]===ltr){
                count++
            }
        }
        return count;
    }

    let answer=[]
    let frequency=0
    let my_dic={}
    for (let i=0; i<lst.length; i++){
        let inst= getCount(lst[i], lst)
        my_dic[lst[i]]=inst
        if (inst >frequency){
            frequency= inst
        }
    }
    for (const i in my_dic){
        if (my_dic[i] === frequency){
            if(isNaN(Number(i))){
                answer.push(i)
            }
            else{
                
                answer.push(Number(i))
            }
        }
    }
    console.log(answer)
    return answer;
}
